src/disco/components/AddonCompatibilityError/style.scss

```diff
@@ -0,0 +1,16 @@
+@import "~ui/css/vars";
+
+.AddonCompatibilityError {
+  background: $report-base-color;
+  font-size: 12px;
+  margin: 0;
+  padding: 5px;
+  text-align: center;
+  width: 100%;
+
+  &,
+  a,
+  a:link {
```

Comment:
The defaults are fine, this just keeps the colour white which it will stay for those other states too.